src/backend/drive/settings.py

```diff
@@ -93,6 +95,10 @@ class Base(Configuration):
     }
     DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+    DATA_DIR = values.Value(
+        os.path.join("/", "data"), environ_name="DJANGO_DATA_DIR", environ_prefix=None
```

Comment:
```suggestion
        os.path.join("/", "data")
```

There is no need to use these arguments if you want to use `DJANGO_DATA_DIR`

Reply:
I also thing `DATA_DIR` should remain a global constant. You should move it after `BASE_DIR` and do this:

```
DATA_DIR = os.getenv("DATA_DIR", os.path.join("/", "data"))
```

Reply:
And the main reason to keep it at the top it's because this is  not a django settings. We reuse it in the settings to compute other settings.